tests/python/codegen/test_target_codegen_aarch64.py

```diff
@@ -476,5 +474,23 @@ def check_correct_assembly(type):
     check_correct_assembly(type=dtype)
 
 
+@pytest.mark.skipif(
+    llvm_version_major() < 10, reason="Vscale is not supported in earlier versions of LLVM"
```

Comment:
Just small nit, 
In ```codegen_llvm.cc``` we have here ```#if TVM_LLVM_VERSION >= 110```, so it would be  ```major < 11``` instead of ```major < 10``` ?
I might be wrong, but being a test case only it is waivable.

Reply:
Thanks, good catch, fixed now!